goroutines/goroutines.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+	"time"
+)
+
+func f(from string) {
+	for i := 0; i < 3; i++ {
+		fmt.Println(from, ":", i)
+	}
+}
+
+func main() {
+	// This run first
+	f("direct")
+	// Goroutines are running concurrently by the Go runtime.
+	go f("goroutine")
+	go func(msg string) {
+		fmt.Println(msg)
+	}("going")
+
+	time.Sleep(time.Second) // Run tne print below with a timelag
```

Comment:
Obviously, for this practice it's OK, but over time I hope you start to associate `time.Sleep` with "Hmm... I'm reducing runtime performance quite a lot here, can I rewrite this to use channels in a better way?".